### ISSUE TITLE
Phase not reflecting altitude configuration

### ISSUE THREAD

**Issue by enjibby** [pnbruckner/ha-sun2#137]
I have set up Sun2 specifically to implement customisations in my household for the mountains to the west. I performed some quick map measurements and placed some values that I thought would reflect the mountain's positions. I noticed that the "Dusk" and "Setting" sensors correctly adjusted their times relative to HA's built-in sun values.

However, the "Phase" sensor still seems to change state later in the evening, matching the HA's built-in sun phase.

I have now modified my automations to do checks for whether "now()" is after the value of the "Setting" sensor (and triggered it every minute) but ideally I'd prefer to just use the "Phase" sensor (or a second, modified sensor) and its state-change event.

**Comment by pnbruckner** [comment]
Yes, in the current implementation, the observer elevation is only used in determining sunrise & sunset and the various dawn & dusk sensors, as well as the daylight & night sensors (which are effectively derived from the dawn & dusk times.)

The phase sensors are based on hard-coded values of the sun's elevation, where sunrise/sunset are defined as -0.833 degrees.

I can see an argument for making the phase sensors depend on observer elevation, but I'll have to think on that some more.

In the meantime, using the Setting sensor is probably best, but you don't have to do it by testing now() every minute. Simply use a time trigger directly with the Setting sensor as defined here: https://www.home-assistant.io/docs/automation/trigger/#sensors-of-datetime-device-class

**Comment by pnbruckner** [comment]
@enjibby I released beta 3.4.2b0 that should fix this issue. Can you give it a try and let me know what you think?
